### PR TITLE
add example of using custom resources

### PR DESCRIPTION
**Pull request by tomplus**
I replace old example of creating third-party resources with an example of managing "custom resources".

Thanks for merging it.

### CODE REVIEW

File: examples/custom_object.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This example use an example CRD from this tutorial:
+# https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/
+#
+# apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (scottilee) [inline examples/custom_object.py]: Can you add a note that you need to apply this YAML?

**Comment by scottilee** [comment]
@tomplus Hey this is awesome apart from my one comment above!

File: examples/custom_object.py (hunk 1 above)
Comment (tomplus) [reply examples/custom_object.py]: added

**Comment by scottilee** [comment]
@yliaog can you take a look?

**Comment by yliaog** [comment]
it's a nice CRD example, thanks.

could you please squash the commits? otherwise looks good

**Comment by tomplus** [comment]
@yliaog squashed, thanks.

**Pull request merged**